specification/postgresql/resource-manager/readme.md

```diff
@@ -30,6 +30,21 @@ description: The Microsoft Azure management API provides create, read, update, a
 openapi-type: arm
 tag: package-2020-01-01
 ```
+### Tag: package-2021-03-25-preview
+
+These settings apply only when `--tag=package-2021-03-25-preview` is specified on the command line.
+
+
+``` yaml $(tag) == 'package-2021-03-25-preview'
+input-file:
+- Microsoft.DBforPostgreSQL/preview/2021-03-25-preview/postgresql.json
+directive:
+  - suppress: R4009
```

Comment:
@aishrk123 - ARM began rolling out the systemData requirement in 2019. At this time, it is required for all RPs and for all resource types. What is your timeline for implementing systemData?